### PR TITLE
{compute-fleet} Releasing customer-facing AZ CLI for compute-fleet version- 2024-11-01

### PR DESCRIPTION
**Pull request by rahuls-microsoft**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/computefleet/tests/latest/test_computefleet.py [hunk 1]
```diff
@@ -1,13 +1,226 @@
...
+            'spot_profile': json.dumps(spot_profile),
+            'regular_priority_profile': json.dumps(regular_profile),
+            'vm_sizes_profile': json.dumps(vm_sizes_profile),
+            'zones': json.dumps(zones),
+            'tags': json.dumps(tags),
+            'tagsNew': json.dumps(tagsNew)
+        })
+
+        try:
+            self.cmd('az computefleet create --name {fleet_name} --resource-group {resource_group} --spot-priority-profile {spot_profile} --compute-profile {compute_profile} --location {location} --tags {tags} ', checks=[
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/computefleet/tests/latest/test_computefleet.py:147]: The name should be `az compute-fleet`

Comment (rahuls-microsoft) [reply src/azure-cli/azure/cli/command_modules/computefleet/tests/latest/test_computefleet.py]: Updated the name to compute-fleet

**Comment by rahuls-microsoft** [comment]
How do we suppress these errors-
- FAIL - HIGH severity: option_length_too_long
    Parameter: compute-fleet fleet create, `additional_locations_profile` - The lengths of all options ['--additional-locations-profile'] are longer than threshold 22. Argument additional_locations_profile must have a short abbreviation.
    Parameter: compute-fleet fleet create, `regular_priority_profile` - The lengths of all options ['--regular-priority-profile'] are longer than threshold 22. Argument regular_priority_profile must have a short abbreviation.
    Parameter: compute-fleet fleet update, `additional_locations_profile` - The lengths of all options ['--additional-locations-profile'] are longer than threshold 22. Argument additional_locations_profile must have a short abbreviation.
    Parameter: compute-fleet fleet update, `regular_priority_profile` - The lengths of all options ['--regular-priority-profile'] are longer than threshold 22. Argument regular_priority_profile must have a short abbreviation.
 
[Pipelines - Run 20241104.28 logs](https://dev.azure.com/azclitools/public/_build/results?buildId=203280&view=logs&j=168ccbe3-da49-5c0b-6478-08f7016e4bf5&t=10a595be-4223-5cc7-9a73-012871b6eaee&l=155) - These ones can ignored, as that's the field name from swagger. Right?

**Comment by rahuls-microsoft** [comment]
How do we supress these errors-
- FAIL - HIGH severity: option_length_too_long
    Parameter: compute-fleet fleet create, `additional_locations_profile` - The lengths of all options ['--additional-locations-profile'] are longer than threshold 22. Argument additional_locations_profile must have a short abbreviation.
    Parameter: compute-fleet fleet create, `regular_priority_profile` - The lengths of all options ['--regular-priority-profile'] are longer than threshold 22. Argument regular_priority_profile must have a short abbreviation.
    Parameter: compute-fleet fleet update, `additional_locations_profile` - The lengths of all options ['--additional-locations-profile'] are longer than threshold 22. Argument additional_locations_profile must have a short abbreviation.
    Parameter: compute-fleet fleet update, `regular_priority_profile` - The lengths of all options ['--regular-priority-profile'] are longer than threshold 22. Argument regular_priority_profile must have a short abbreviation.
 
[Pipelines - Run 20241104.28 logs](https://dev.azure.com/azclitools/public/_build/results?buildId=203280&view=logs&j=168ccbe3-da49-5c0b-6478-08f7016e4bf5&t=10a595be-4223-5cc7-9a73-012871b6eaee&l=155) - These ones can ignored, as that's the field name. Right?

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**